backend/api/Controllers/AuthController.cs

```diff
@@ -89,14 +67,36 @@ public IActionResult Activate()
         }
 
         /// <summary>
-        /// Log the current user out.
+        /// Redirect to the keycloak login page.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("[action]")]
+        public IActionResult Login(string redirect_uri)
+        {
+            var loginUrl = $"{_configuration.GetValue<string>("Keycloak:Authority")}{_configuration.GetValue<string>("Keycloak:OpenIdConnect:Login")}";
```

Comment:
I think I preferred the error handling pattern in the other controller to get these values.